packages/client/src/forms/configuration/index.ts

```diff
@@ -82,7 +82,7 @@ function mergeIdentifiers(
   return [
     ...mergedIdentifiers.filter(
       ({ sectionIndex, groupIndex }) =>
-        sectionIndex !== questionIdentifiers.sectionIndex &&
+        sectionIndex !== questionIdentifiers.sectionIndex ||
```

Comment:
The actual conditional is
```ts
!(sectionIndex === questionIdentifiers.sectionIndex && groupIndex === questionIdentifiers.groupIndex)
```
I had forgotten to negate the `&&` to `||` when negating the whole expression